<launch>
	<arg name="board_frame" default="ar_marker_0" />

	<node name="object_masker" pkg="object_pose_utils" type="object_masker.py" ns="object_masker" output="screen">
        <param name="board_frame" value="$(arg board_frame)" />
        <remap from="in_image" to="/image_folder/image" />
        <remap from="in_camera_info" to="/image_folder/camera_info" />
    </node>
	
</launch>
